.github/workflows/test-convene-web.yml

```diff
@@ -172,3 +172,19 @@ jobs:
         with:
           name: feature-test-failed-screenshot
           path: features/test_reports/*.png
+
+  lint:
+    name: Run style checks
+    runs-on: ubuntu-latest
+    needs: [setup]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Ruby and install gems
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - run: bundle exec rubocop --parallel --config .rubocop.yml
```

Comment:
Do we want one for `prettier` as well? That's what is (ostensibly) formatting our `css` and `js`?

Reply:
Added.